src/tag/styled-components.js

```diff
@@ -0,0 +1,60 @@
+/*
+Copyright (c) 2018 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+import {styled, hexToRgb} from '../styles/index';
+
+export const Action = styled('div', props => {
+  const {$color, $disabled} = props;
+  return {
+    display: 'flex',
+    alignItems: 'center',
+    padding: '12px',
+    borderTopRightRadius: '2px',
+    borderBottomRightRadius: '2px',
+    marginLeft: '8px',
+    cursor: $disabled ? 'not-allowed' : 'pointer',
+    ':hover': {
+      backgroundColor: $disabled ? null : hexToRgb($color, '0.2'),
+    },
+  };
+});
+
+export const ActionIcon = styled('svg', () => {
+  return {};
+});
+
+export const Root = styled('span', props => {
+  const {$theme, $color, $disabled, $isFocused, $isActive} = props;
+  const {
+    sizing: {scale800, scale100, scale300},
+    typography: {font200},
+  } = $theme;
+  return {
+    ...font200,
+    display: 'flex',
```

Comment:
Should we use inline-flex? Tags seem like more of an inline element than a block-level element (I'm assuming that's why they're a span too)

Reply:
ok